logs.txt

```diff
@@ -0,0 +1,7 @@
+ * Serving Flask app "todo_app/app" (lazy loading)
```

Comment:
This should be added to .gitignore

And you should delete the `.venv.old` folder! There's no need to back it up, you can always recreate it with `poetry install` if you switch back to `in-project = true`

Reply:
deleted the .venv.old folder and added logs.txt to .gitignore